src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -668,25 +668,21 @@ private HttpNamedRequestNameNode ParseNamedRequestNameNode()
                 node.AddDiagnostic(diagnostic);
             }
             bool wordParsedOnce = false;
-            while (MoreTokens() && CurrentToken is not {Kind: TokenKind.NewLine })
-                {
+            while (MoreTokens() && CurrentToken is not { Kind: TokenKind.NewLine })
+            {
 
-                    if (CurrentToken is not null && (!(CurrentToken is { Kind: TokenKind.Word or TokenKind.Whitespace} or { Text: "_" or "@" or "."}) || CurrentToken is {Kind: TokenKind.Word } && wordParsedOnce)) 
-                    {
-                        var diagnostic = CurrentToken.CreateDiagnostic(HttpDiagnostics.InvalidNamedRequestName());
-                        node.AddDiagnostic(diagnostic);
-                        wordParsedOnce = false;
-                    }
+                if (CurrentToken is not null && (!(CurrentToken is { Kind: TokenKind.Word or TokenKind.Whitespace } or { Text: "_" or "@" or "." }) || CurrentToken is { Kind: TokenKind.Word } && wordParsedOnce))
+                {
+                    var diagnostic = CurrentToken.CreateDiagnostic(HttpDiagnostics.InvalidNamedRequestName());
+                    node.AddDiagnostic(diagnostic);
+                    wordParsedOnce = false;
+                }
 
-                    if (CurrentToken is { Kind: TokenKind.Word })
-                    {
-                        wordParsedOnce = true;
-                    }
-                    ConsumeCurrentTokenInto(node);
-                /*if (CurrentToken is { Kind: TokenKind.Whitespace })
+                if (CurrentToken is { Kind: TokenKind.Word })
                 {
-                    ParseTrailingWhitespace(node, stopBeforeNewLine: true);
-                }*/
```

Comment:
Was this commented for testing something? If yes, please revert before merge. Otherwise, we should probably delete the commented code.

Reply:
It wasn't testing anything, I just noticed it was left over from last time and wanted to remvove it

Reply:
Ah please remove the commented block then to avoid any confusion